### PR TITLE
Added adapter for Date::Holidays::BY

### PR DESCRIPTION
**Pull request by bes-internal**

### CODE REVIEW

File: README.mkdn [hunk 1]
```diff
@@ -462,6 +462,9 @@ Test coverage in version 1.00
 - [Date::Holidays::AT](https://metacpan.org/pod/Date::Holidays::AT)
 - [Date::Holidays::CN](https://metacpan.org/pod/Date::Holidays::CN)
 - [Date::Holidays::RU](https://metacpan.org/pod/Date::Holidays::RU)
+- [Date::Holidays::Adapter::NO](https://metacpan.org/pod/Date::Holidays::Adapter::RU)
```
Comment (jonasbn) [inline README.mkdn]: This seems wrong, it mentions Norway (NO)

File: Changes [hunk 2]
```diff
@@ -1,5 +1,10 @@
 Changes file for Date-Holidays
 
+1.04 2017-05-30 Feature release, update not required
```
Comment (jonasbn) [inline Changes]: I am not sure the release will go out today, but as soon as possible :-)

File: lib/Date/Holidays/Adapter/BY.pm [hunk 3]
```diff
@@ -0,0 +1,121 @@
+package Date::Holidays::Adapter::BY;
+
+use strict;
+use warnings;
+use vars qw($VERSION);
+
+use base 'Date::Holidays::Adapter';
+
+$VERSION = '1.03';
```
Comment (jonasbn) [inline lib/Date/Holidays/Adapter/BY.pm]: This version number has to reflect the distribution version number so package/distribution relation is clean, so it should read 1.04

File: Changes (hunk 2 above)
Comment (bes-internal) [inline Changes]: You need to revise this format, if not only you are involved in the development

File: README.mkdn (hunk 1 above)
Comment (bes-internal) [inline README.mkdn]: How did I miss this... Thanks!

File: lib/Date/Holidays/Adapter/BY.pm (hunk 3 above)
Comment (bes-internal) [inline lib/Date/Holidays/Adapter/BY.pm]: agree, done

File: Changes (hunk 2 above)
Comment (jonasbn) [inline Changes]: @bes-internal I am not sure what you mean, care to elaborate?

Comment (bes-internal) [inline Changes]: I just wanted to say that this date will always be inaccurate if the pull request are waiting for long enough

**Comment by jonasbn** [comment]
Ah I understand, well I do not expect for contributors to make releases, at least it requires co-maintainership on PAUSE for upload to CPAN.

Thanks for clarifying

**Comment by jonasbn** [comment]
@bes-internal I think you have to push your changes again, or create a new PR, I merged based on your review feedback, but the changes does not seem to be present

**Comment by jonasbn** [comment]
@bes-internal nevermind, I worked it out, the README is generated, not maintained by hand

**Comment by bes-internal** [comment]
@jonasbn You will finish list in the pod?

**Comment by jonasbn** [comment]
Yes, working on building the release now

**Pull request merged**